command/commands/team.py

```diff
@@ -476,3 +486,59 @@ def delete_helper(self, team_name, user_id) -> ResponseTuple:
             logging.error("team delete unsuccessful")
             return f"Team delete was unsuccessful with " \
                    f"the following error: {e.data}", 200
+
+    def refresh_helper(self, user_id) -> ResponseTuple:
+        """
+        Ensure that the local team database is the same as GitHub's.
+
+        In the event that our local team database is outdated compared to
+        the teams on GitHub, this command can be called to fix things.
+        :return: return error message if user has insufficient permission level
+                 otherwise returns success messages with # of teams changed
+        """
+        num_changed = 0
+        num_added = 0
+        num_deleted = 0
+        modified = []
+        try:
+            command_user = self.facade.retrieve(User, user_id)
+            if not check_permissions(command_user, None):
+                return self.permission_error, 200
+            local_teams = self.facade.query(Team)
+            remote_teams = self.gh.org_get_teams()
+            local_ids = dict((team.github_team_id, team)
+                             for team in local_teams)
+            remote_ids = dict((team.github_team_id, team)
+                              for team in remote_teams)
+
+            # remove teams not in github anymore
+            for local_id in local_ids:
+                if local_id not in remote_ids:
+                    self.gh.org_delete_team(local_id)
+                    num_deleted += 1
+                    modified.append(local_ids[local_id].get_attachment())
+
+            # add teams to db that are in github but not in local database,
+            for remote_id in remote_ids:
+                if remote_id not in local_ids:
+                    self.facade.store(remote_ids[remote_id])
```

Comment:
You should create a new team object here and store it, since the `remote_id` teams aren't Teams from our model, but PyGithub Team objects. Sorry for the confusion :/

Reply:
Ty! Done